pkg/clicmd/env_set.go

```diff
@@ -41,6 +42,7 @@ var envSetCmd = &cobra.Command{
 			actions.OptionEnvName:    args[0],
 			actions.OptionNewEnvName: viper.GetString(vEnvSetName),
 			actions.OptionNamespace:  viper.GetString(vEnvSetNamespace),
+			actions.OptionServer:     viper.GetString(vEnvSetServer),
```

Comment:
If you add a new option, you'll have to adjust the test to expect the new option.

Reply:
add tests for both clicmd and actions